airflow/providers/google/cloud/transfers/oracle_to_gcs.py

```diff
@@ -45,15 +45,15 @@ class OracleToGCSOperator(BaseSQLToGCSOperator):
     ui_color = '#a0e08c'
 
     type_map = {
-        cx_Oracle.DB_TYPE_BINARY_DOUBLE: 'DECIMAL',
-        cx_Oracle.DB_TYPE_BINARY_FLOAT: 'DECIMAL',
-        cx_Oracle.DB_TYPE_BINARY_INTEGER: 'INTEGER',
-        cx_Oracle.DB_TYPE_BOOLEAN: 'BOOLEAN',
-        cx_Oracle.DB_TYPE_DATE: 'TIMESTAMP',
-        cx_Oracle.DB_TYPE_NUMBER: 'NUMERIC',
-        cx_Oracle.DB_TYPE_TIMESTAMP: 'TIMESTAMP',
-        cx_Oracle.DB_TYPE_TIMESTAMP_LTZ: 'TIMESTAMP',
-        cx_Oracle.DB_TYPE_TIMESTAMP_TZ: 'TIMESTAMP',
+        oracledbbase.DB_TYPE_BINARY_DOUBLE: 'DECIMAL',
+        oracledbbase.DB_TYPE_BINARY_FLOAT: 'DECIMAL',
+        oracledbbase.DB_TYPE_BINARY_INTEGER: 'INTEGER',
+        oracledbbase.DB_TYPE_BOOLEAN: 'BOOLEAN',
+        oracledbbase.DB_TYPE_DATE: 'TIMESTAMP',
+        oracledbbase.DB_TYPE_NUMBER: 'NUMERIC',
+        oracledbbase.DB_TYPE_TIMESTAMP: 'TIMESTAMP',
+        oracledbbase.DB_TYPE_TIMESTAMP_LTZ: 'TIMESTAMP',
+        oracledbbase.DB_TYPE_TIMESTAMP_TZ: 'TIMESTAMP',
```

Comment:
I do not recommend direct using underlay implementation of any package. Is there any reason why not using:
```python
import oracledb

oracledb.DB_TYPE_TIMESTAMP_TZ: 'TIMESTAMP'
```

Reply:
I had that originally, but MyPy didn't like that. I'd get the following error:
`error: Module has no attribute DB_TYPE_TIMESTAMP_TZ: 'TIMESTAMP'`. This was the only way I could find to stop the error.

Reply:
@potiuk, I feel like my solution here is a little bit hacky. The only problem I have with doing it the way @dungdm93 suggest is that it causes issues with MyPy, stating the error I wrote above. I did try the fix you mentioned in Slack (`breeze build-image --upgrade-to-newer-depenencies --python 3.7`) but that did not solve the issue.

You also suggested for another issue I had with MyPy that some times ignoring a specific typing error is the easiest approach. Do you think this might also be the case for the issue?